### PR TITLE
topology: add jsl nocodec topology

### PR DESCRIPTION
**Pull request by xiulipan**
Add sof-jsl-nocodec topology from sof-icl-nocodec.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>

### CODE REVIEW

File: tools/topology/sof-icl-nocodec.m4 [hunk 1]
```diff
@@ -14,8 +14,8 @@ include(`common/tlv.m4')
 # Include Token library
 include(`sof/tokens.m4')
 
-# Include Icelake DSP configuration
-include(`platform/intel/icl.m4')
+# Include DSP configuration
+include(`platform/intel/'PLATFORM`.m4')
```
Comment (paulstelian97) [inline tools/topology/sof-icl-nocodec.m4:18]: I assume `platform/intel/jsl.m4` does exist (can't check right now), right?

Comment (xiulipan) [reply tools/topology/sof-icl-nocodec.m4:18]: Why not exist? We already have a lot of sof-jsl* tplg

**Review by lgirdwood — APPROVED** [review]

File: tools/topology/sof-icl-nocodec.m4 (hunk 1 above)
Comment (plbossart) [reply tools/topology/sof-icl-nocodec.m4:18]: >
a better answer:

````bash
ls platform/intel/jsl*
platform/intel/jsl-dedede.m4  platform/intel/jsl.m4  platform/intel/jsl-rt1015.m4
````

**Comment by lgirdwood** [comment]
Jekins CI showing a DUT HW failure on BSW.

**Pull request merged**